bin/crew

```diff
@@ -210,33 +222,59 @@ def list_installed
 end
 
 def list_compatible(compat = true)
+  generate_compatible
```

Comment:
Is this necessary every time `crew list compatible` is run?  I'm thinking it's only needed once during `crew update`.

Reply:
I refactored `list_compatible` to not duplicate code with `generate_compatible`. Now the read of all package files happens during generate_compatible,which then outputs into the JSON device file. `list_compatible` now just reads the JSON file.

The entire process takes about 3 seconds.
crew update compatible:
```
real    0m1.490s
user    0m1.405s
sys     0m0.085s
```

crew list compatible:
```
real    0m2.072s
user    0m1.968s
sys     0m0.104s
```

One could just read the json file, but in case you have updated local packages, I think it makes sense to spend the extra 1.5 seconds to check to see if those changes fixed anything?

Reply:
It takes 3 seconds in your Docker container but what about the many underpowered Chromebooks in existence?